packages/patternfly-4/react-integration/demo-app-ts/package.json

```diff
@@ -7,7 +7,7 @@
     "start:demo-app": "react-scripts start"
   },
   "dependencies": {
-    "@patternfly/react-core": "file:../../react-core",
+    "@patternfly/react-core": "^3.16.7",
```

Comment:
This should be built against what you are building right now, and not a release.  The goal is to make sure the code you add for a component will work in an app.